### ISSUE TITLE
Black screen when opening Etcher

### ISSUE THREAD

**Issue by flightless-katie** [balena-io/etcher#3057]
- **Etcher version:** 1.5.74
- **Operating system and architecture:** Mac OS Mojave 10.14.6
- **Image flashed:** N/A
- **Do you see any meaningful error information in the DevTools?** Yes, see below

After tracing the error as the program is running, I found that in leds.ts this code seems to be throwing: (this is on startup, first time trying Etcher)
```
const ledsMapping = settings.get('ledsMapping');
for (const [drivePath, ledsPaths] of Object.entries(ledsMapping)) { // <-- this line throws
  leds.set('/dev/disk/by-path/' + drivePath, new RGBLed(ledsPaths));
}
```
settings.get(...) is returning undefined, so when the for loop tries to use Object.entries(ledsMapping), it is trying to get the entries of undefined, which causes an error.

If in devtools I set ledsMapping to an empty object `{}` the program continues to the opening menu to select image, etc. It's doing a flash operation now with that manual hack. :)

**Comment by flightless-katie** [comment]
PS, I am not familiar with your codebase or any conventions you may have, but this could be a potential fix, maybe?
```
const ledsMapping = settings.get('ledsMapping') || {};
```

**Comment by barrypress** [comment]
This did work, but not being familiar with the codebase or the dev tools, I had to fumble around a lot before I finally set enough breakpoints (discovering cmd-R was a biggie, too) and pasted in the patch in enough files. I believe I had to do at least gui.js and leds.ts, but not altogether sure which one worked.

However, the fix is not persistent - closing and reopening the pap results in the same gray useless window.
